src/cljs/velho_ds/organisms/heading.cljs

```diff
@@ -2,35 +2,58 @@
   (:require [stylefy.core :as stylefy]
             [clojure.string :as string]
             [velho-ds.organisms.style.heading :as style]
+            [velho-ds.tokens.color :as color]
             [velho-ds.tokens.font :as fonts]
             [velho-ds.tokens.font-size :as font-sizes]
             [velho-ds.atoms.icon :as icons]
+            [velho-ds.atoms.area :as areas]
+            [velho-ds.atoms.button :as buttons]
             [velho-ds.molecules.field :as fields]
+            [velho-ds.molecules.tab :as tabs]
             [velho-ds.organisms.grid :as grid]
+            [velho-ds.tools.style :as tools-style]
             [reagent.core :as r]))
 
 (defn main [content]
   [:header.header-style
    [:h2.heading-style content]])
 
-(defn content-heading [{:keys [status info actions footer]}]
-  [grid/grid-wrap {:rows 3
-                   :cols 3
-                   :styles style/content-heading}
-   (when status
-     (into [grid/grid-cell {:col-start 1, :col-end 4, :styles {:padding-bottom "1px"}}]
-           (for [item status] ^{:key item} item)))
-   [grid/grid-cell {:col-start 1, :col-end 4, :styles style/content-heading-info}
-    [:p (stylefy/use-style style/content-heading-info-item) (get info :type)]
-    [:h1 (stylefy/use-style style/content-heading-info-item) (get info :name)]
-    [:h2 (stylefy/use-style style/content-heading-info-item) (get info :footnote)]]
-   (when actions
-     (into [grid/grid-cell {:col-start 4, :col-end 4, :styles style/content-heading-actions}]
-           (for [item actions] ^{:key item} item)))
-   [grid/grid-cell {:col-start 1, :col-end 4}
-    (when footer
-      (into [:ul (stylefy/use-style style/content-heading-footer)]
-            (for [item footer] ^{:key item} [:li (stylefy/use-sub-style style/content-heading-footer :li) item])))]])
+
+(defn content-info [{:keys [breadcrumb footnote meta navigation bar-color breadcrumb-click-fn styles]}]
```

Comment:
I'm not so sure if the key-naming is suitable for general use here.

Reply:
Lets discuss this and change if needed